### ISSUE TITLE
ideas for center pivot detection, improving accuracy in general

### ISSUE THREAD

**Issue by rbavery** [rbavery/CropMask_RCNN#5]
smaller steps_per_epoch in config. Currently set way above Number of samples/Batch size, which means in an epoch the training set is passed through many times over (not a true epoch, where training set goes through once). this means validation loss is probably not checked often enough as a stopping condition and I may be missing good local minima within epochs.

more negative samples in center pivot detection

image augmentation, start with simple flips and add rotates, gaussian blur, etc.

increase batch size to 3? 4 didn't work but 3 might

clean up bad masks of conjoined center pivots/small ag to just have center pivots
